service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/cmo/CMOReadyToSealEventHandlerTest.java

```diff
@@ -0,0 +1,167 @@
+package uk.gov.hmcts.reform.fpl.handlers.cmo;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.events.cmo.CMOReadyToSealEvent;
+import uk.gov.hmcts.reform.fpl.handlers.HmctsAdminNotificationHandler;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.HearingBooking;
+import uk.gov.hmcts.reform.fpl.model.Judge;
+import uk.gov.hmcts.reform.fpl.model.Respondent;
+import uk.gov.hmcts.reform.fpl.model.RespondentParty;
+import uk.gov.hmcts.reform.fpl.model.common.JudgeAndLegalAdvisor;
+import uk.gov.hmcts.reform.fpl.model.event.EventData;
+import uk.gov.hmcts.reform.fpl.model.notify.cmo.CMOReadyToSealTemplate;
+import uk.gov.hmcts.reform.fpl.service.email.NotificationService;
+import uk.gov.hmcts.reform.fpl.service.email.content.cmo.CMOReadyToSealContentProvider;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyMap;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.fpl.NotifyTemplates.CMO_READY_FOR_JUDGE_REVIEW_NOTIFICATION_TEMPLATE;
+import static uk.gov.hmcts.reform.fpl.NotifyTemplates.CMO_READY_FOR_JUDGE_REVIEW_NOTIFICATION_TEMPLATE_JUDGE;
+import static uk.gov.hmcts.reform.fpl.enums.JudgeOrMagistrateTitle.HIS_HONOUR_JUDGE;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
+
+@ExtendWith(MockitoExtension.class)
+class CMOReadyToSealEventHandlerTest {
+
+    private static final String HMCTS_ADMIN_EMAIL = "admin@hmcts.gov.uk";
+    private static final String HMCTS_JUDGE_EMAIL = "judge@hmcts.gov.uk";
+
+    @Mock
+    private NotificationService notificationService;
+
+    @Mock
+    private HmctsAdminNotificationHandler adminNotificationHandler;
+
+    @Mock
+    private CMOReadyToSealContentProvider contentProvider;
+
+    @Mock
+    private ObjectMapper mapper;
+
+    @InjectMocks
+    private CMOReadyToSealEventHandler eventHandler;
+
+    @Test
+    void shouldSendNotificationForAdmin() {
+        CaseData caseData = caseData(false);
```

Comment:
As mentioned somewhere else I would probably expect two methods: (which can call this one with true / false). caseDataWithJudge and without. Such a small thing and increases the readability of the tests, now I need to jump below to understand what's going on.